.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,17 @@
+name: golangci-lint
+on:
+  push:
+  pull_request:
```

Comment:
this fixes the linting on forks/branches so devs won't need to create a PR in order for the builds and linters to run.